src/privacy_compliance.js

```diff
@@ -78,17 +76,34 @@ class PrivacyCompliance {
     return this.supportedCapabilities.has(capability);
   }
 
+  hasFrameworkLoadedToGenerate(ability) {
```

Comment:
Would `canGenerate` be a more succinct name? Took me a few reads of the method name to realize what it was going for. Again I could still be wrong.

Reply:
very nice change 👍